docs/tutorials/basics/60_methods.md

```diff
@@ -107,3 +107,60 @@ say_hello 3
 
 Overloading isn't defined just by type restrictions. The number of arguments as well as named arguments are also
 relevant characteristics.
+
+## Return value
+
+Methods may also return a value:
+
+```crystal-play
+def adds_2(n : Int32 = 0)
```

Comment:
What's the point of the method having a default argument?

Reply:
I'm pretty sure that is a leftover of another version of the example 🙈